Samples/3a-literate-swagger/generated/swagger.md.json

```diff
@@ -49,7 +48,8 @@
             "$ref": "#/parameters/ResourceGroupName"
           },
           {
-            "$ref": "#/parameters/SearchServiceName"
+            "$ref": "#/parameters/SearchServiceName",
+            "description": "> Search is also performed relative to nodes discovered by parent headings.\n>\n> In this case, we're in the scope of the node identified by `#QueryKeys_List` which is the operation.\n>\n> Furthermore, `#SearchServiceName` is really shorthand for `@.#SearchServiceName` where `@` represents the current scope's node.\n> One could force searching the global scope with `$.#SearchServiceName`\n\nSome description."
```

Comment:
that one runs on the code model (it was simpler that way) :/ we may indeed wanna change that since other endpoints may benefit from stripping GFM (but not force that, otherwise our "swagger resolver" endpoint may do unexpected magic to some people)